solutions/instances/main.tf

```diff
@@ -202,3 +208,47 @@ module "scc_wp" {
   access_tags                   = var.scc_workload_protection_access_tags
   scc_wp_service_plan           = var.scc_workload_protection_service_plan
 }
+
+#######################################################################################################################
+# SCC Event Notifications Configuration
+#######################################################################################################################
+
+locals {
+  parsed_existing_en_instance_crn = var.existing_en_crn != null ? split(":", var.existing_en_crn) : []
+  existing_en_guid                = length(local.parsed_existing_kms_instance_crn) > 0 ? local.parsed_existing_kms_instance_crn[7] : null
+}
+
+data "ibm_en_destinations" "en_destinations" {
+  count         = var.existing_en_crn != null ? 1 : 0
+  instance_guid = var.existing_en_guid
+}
+
+resource "ibm_en_topic" "en_topic" {
+  count         = var.existing_en_crn != null ? 1 : 0
```

Comment:
should we not support creating the topic in an existing instance?

Reply:
if they provide an existing_en_crn that is not null, it will create an en topic in that existing_en_crn instance, else the resource will not be created.

Reply:
you need an existing topic and subscription to enable any kind of notification from event notifications.

Reply:
ah sorry my bad - I was thinking we created EN in this repo, but we always take in an existing instance duh